Unable to execute use SecretsManagerConfigProvider on Release 0.3.0

I'm using msk-config-provider with SecretsManagerConfigProvider to send data to Aurora using confluentinc-kafka-connect-jdbc from MSK Connect

I'm able to use it using the Release 0.2.0, nevertheless, on the Release 0.3.0 the connector is not able to start because the following errors:

```
2024-12-02T13:37:37.000Z
[Worker-0dc6bb008ecab2cda] [2024-12-02 13:37:37,325] INFO SecretsManagerConfig values:
2024-12-02T13:37:37.000Z
[Worker-0dc6bb008ecab2cda] NotFoundStrategy = ignore
2024-12-02T13:37:37.000Z
[Worker-0dc6bb008ecab2cda] endpoint =
2024-12-02T13:37:37.000Z
[Worker-0dc6bb008ecab2cda] region = eu-west-1
2024-12-02T13:37:37.000Z
[Worker-0dc6bb008ecab2cda] (com.amazonaws.kafka.config.providers.SecretsManagerConfig:361)
2024-12-02T13:37:37.000Z
[Worker-0dc6bb008ecab2cda] [2024-12-02 13:37:37,327] WARN /connectors (org.eclipse.jetty.server.HttpChannel:776)
2024-12-02T13:37:37.000Z
[Worker-0dc6bb008ecab2cda] javax.servlet.ServletException: org.glassfish.jersey.server.ContainerException: java.lang.NoSuchFieldError: CLIENT_ENDPOINT_PROVIDER
2024-12-02T13:37:37.000Z
[Worker-0dc6bb008ecab2cda] at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:410)

....

2024-12-02T13:37:31.000Z
[Worker-0dc6bb008ecab2cda] ... 39 more
2024-12-02T13:37:31.000Z
[Worker-0dc6bb008ecab2cda] Caused by: java.lang.NoSuchFieldError: CLIENT_ENDPOINT_PROVIDER
2024-12-02T13:37:31.000Z
[Worker-0dc6bb008ecab2cda] at software.amazon.awssdk.services.secretsmanager.DefaultSecretsManagerBaseClientBuilder.finalizeServiceConfiguration(DefaultSecretsManagerBaseClientBuilder.java:101)
2024-12-02T13:37:31.000Z
[Worker-0dc6bb008ecab2cda] at software.amazon.awssdk.awscore.client.builder.AwsDefaultClientBuilder.finalizeChildConfiguration(AwsDefaultClientBuilder.java:160)
2024-12-02T13:37:31.000Z
[Worker-0dc6bb008ecab2cda] at software.amazon.awssdk.core.client.builder.SdkDefaultClientBuilder.syncClientConfiguration(SdkDefaultClientBuilder.java:188)
2024-12-02T13:37:31.000Z
[Worker-0dc6bb008ecab2cda] at software.amazon.awssdk.services.secretsmanager.DefaultSecretsManagerClientBuilder.buildClient(DefaultSecretsManagerClientBuilder.java:38)
2024-12-02T13:37:31.000Z
[Worker-0dc6bb008ecab2cda] at software.amazon.awssdk.services.secretsmanager.DefaultSecretsManagerClientBuilder.buildClient(DefaultSecretsManagerClientBuilder.java:25)
2024-12-02T13:37:31.000Z
[Worker-0dc6bb008ecab2cda] at software.amazon.awssdk.core.client.builder.SdkDefaultClientBuilder.build(SdkDefaultClientBuilder.java:155)
2024-12-02T13:37:31.000Z
[Worker-0dc6bb008ecab2cda] at com.amazonaws.kafka.config.providers.SecretsManagerConfigProvider.checkOrInitSecretManagerClient(SecretsManagerConfigProvider.java:175)
2024-12-02T13:37:31.000Z
[Worker-0dc6bb008ecab2cda] at com.amazonaws.kafka.config.providers.SecretsManagerConfigProvider.get(SecretsManagerConfigProvider.java:137)
```

--- a/pom.xml
+++ b/pom.xml
@@ -2,7 +2,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>software.amazon.msk</groupId>
 	<artifactId>msk-config-providers</artifactId>
-	<version>0.3.1</version>
+	<version>0.4.0-SNAPSHOT</version>
 	<name>msk-config-providers</name>
 	<description>custom config plugins for Kafka Connect</description>
 

--- a/pom.xml
+++ b/pom.xml
@@ -2,7 +2,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>software.amazon.msk</groupId>
 	<artifactId>msk-config-providers</artifactId>
-	<version>0.4.0-SNAPSHOT</version>
+	<version>0.3.1</version>
 	<name>msk-config-providers</name>
 	<description>custom config plugins for Kafka Connect</description>
 
@@ -12,7 +12,7 @@
 		<maven.compiler.target>${maven.compiler.source}</maven.compiler.target>
 		<junit.jupiter.version>5.9.1</junit.jupiter.version>
 		<slf4j.version>1.7.32</slf4j.version>
-		<software.amazon.awssdk.version>2.29.11</software.amazon.awssdk.version>
+		<software.amazon.awssdk.version>2.18.8</software.amazon.awssdk.version>
 	</properties>
 
 	<dependencies>

--- a/pom.xml
+++ b/pom.xml
@@ -2,7 +2,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>software.amazon.msk</groupId>
 	<artifactId>msk-config-providers</artifactId>
-	<version>0.3.0</version>
+	<version>0.4.0-SNAPSHOT</version>
 	<name>msk-config-providers</name>
 	<description>custom config plugins for Kafka Connect</description>
 

